spec/controllers/users_spec.rb

```diff
@@ -0,0 +1,131 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+  describe "create" do
+    let(:user) {User.create(:first_name => "Rebecca", :last_name => "Holzschuh", :email => "rebeccaholzschuh@gmail.com", :username=> "rebecca")}
+    it "adds one user to the database" do
+      expect {user}.to change(User, :count).by(1)
+    end
+
+    it "adds the user to the database" do
+      User.create(:first_name => "Rebecca")
+      expect(User.last.first_name).to eq("Rebecca")
+    end
+
+    it "redirects back to the home page after saving a user" do
+      post :create, user: {:first_name => "Rebecca", :last_name => "Holzschuh"}
+      expect(response).to redirect_to root_path
+    end
+  end
+
+  before(:each) do
+    @user = User.create(:first_name => "Rebecca", :last_name => "Holzschuh", :email => "rebeccaholzschuh@gmail.com", :username=> "rebecca")
+  end
+
+  describe "show" do
+    subject {get :show}
+    it "renders show template" do
+      expect(subject).to render_template(:show)
+    end
+  end
+
+  describe 'trim' do
+    it 'user starts at trimmed = false' do
+      expect(@user.trimmed).to be false
+    end
+
+    it 'trims a user' do
+      put :trim, :id => @user.id
```

Comment:
Use the expect { ... }.to change(...).to(...) to DRY up this test and the ones following.

https://www.relishapp.com/rspec/rspec-expectations/v/2-0/docs/matchers/expect-change

example...
```
it "trims a user" do
   expect {
     put :trim, id: @user.id
  }.to change(@user.trimmed).from(false).to(true)
end
```

That will encompass everything you are trying to test for both examples.  I suggest you apply it to all your tests below this as well.